composer.json

```
@@ -22,6 +22,11 @@
         "drupal/core-project-message": "^11.2",
         "drupal/core-recipe-unpack": "^11.2",
         "drupal/core-recommended": "^11.2",
+        "drupal/externalauth": "^2",
```

```suggestion
        "drupal/externalauth": "2",
```

Can we update the packages in here to avoid using automatic updates like in the suggestion provided? We have a policy of pinning dependencies due to a few security issues we've seen with malicious package updates, so prefer to keep them static and have a developer manually update the packages.

---

This has been updated. Please check and let me know. 
Thanks.